src/typed-method-types/workflows/triggers/inputs.ts

```diff
@@ -63,7 +78,7 @@ type WorkflowInputsType<Params extends InputParameterSchema> =
  */
 export type WorkflowInputs<WorkflowDefinition extends WorkflowSchema> =
   WorkflowDefinition["title"] extends NO_GENERIC_TITLE ? {
-      /** @description The inputs provided to the workflow */
+      /** @description Either `value` or `customizable` should be provided, but not both. */
```

Comment:
I think we should keep the note that this is passing values to the workflow's inputs.

```suggestion
      /** @description The inputs provided to the workflow. Either `value` or `customizable` should be provided, but not both. */
```